packages/core/src/scripts/prisma.ts

```diff
@@ -1,12 +1,18 @@
 import execa from 'execa';
 import { createSystem } from '../lib/createSystem';
 import { generateNodeModulesArtifacts, validateCommittedArtifacts } from '../artifacts';
-import { loadConfigOnce } from '../lib/config/loadConfig';
+import { loadConfigOnce, loadBuiltConfig } from '../lib/config/loadConfig';
+import { KeystoneConfig } from './../types/config/index';
+
 import { ExitError } from './utils';
 
 export async function prisma(cwd: string, args: string[]) {
-  const config = await loadConfigOnce(cwd);
-
+  let config: KeystoneConfig;
+  if (process.env.NODE_ENV === 'production') {
+    config = loadBuiltConfig(cwd);
+  } else {
+    config = await loadConfigOnce(cwd);
+  }
```

Comment:
Use `--frozen` for this, so we are consistent.
Unexpected behaviour from `NODE_ENV` is not fun